doc/cmdline.rst

```diff
@@ -139,6 +139,19 @@ Environment variables
    See :ref:`uploading packages with environment variables <upload_envvars>`
    for more information.
 
+.. envvar:: FLIT_ALLOW_INVALID
```

Comment:
`FLIT_ALLOW_INVALID_METADATA` ? You may want invalid other things later on ?

Reply:
I think I'll live with that risk. I'm not sure that the scope of 'metadata' is very clear, e.g. entry points are not part of the `[tool.flit.metadata]` table.

This is meant to be an escape hatch for when a check is wrong, so I don't think it needs to be fine-grained.

Reply:
I'm ok with it, it was mostly a question. You're the boss, boss.

Reply:
![who's the boss](http://pic.eslite.com/Upload/Product/201507/m/635720471352566250.jpg)

It was a point worth thinking about :-)